tests/pkg_tinydtls_sock_async/dtls-server.c

```diff
@@ -117,10 +117,41 @@ static void _dtls_handler(sock_dtls_t *sock, sock_async_flags_t type, void *arg)
         puts("New client connected");
     }
     if (type & SOCK_ASYNC_CONN_FIN) {
-        puts("Session was destroyed by peer");
+        if (sock_dtls_get_event_session(sock, &session))
+        {
```

Comment:
minor nit: put the braces on the same line as the if statement

Reply:
fixed! And also uncluttered the DTLS handler now